src/pip/_internal/metadata/base.py

```diff
@@ -521,6 +521,11 @@ def iter_installed_distributions(
     ) -> Iterator[BaseDistribution]:
         """Return a list of installed distributions.
 
+        This is based on ``iter_all_distributions()``, but applies some
+        configurabilities. Note that ``iter_installed_distributions()`` without
```

Comment:
configurabilities -> filtering?

Reply:
That’s a reasonable choice of word, yes.